src/tests/sqlsmith/src/runner.rs

```diff
@@ -548,51 +548,53 @@ async fn diff_stream_and_batch(
     // Generate some mviews
     let mview_name = format!("stream_{}", i);
     let (batch, stream, table) = differential_sql_gen(rng, mvs_and_base_tables, &mview_name)?;
+    diff_stream_and_batch_with_sqls(client, i, &batch, &stream, &mview_name, &table).await
+}
 
-    tracing::info!("[EXECUTING DIFF - CREATE MVIEW id={}]: {}", i, &stream);
-    let skip_count = run_query(12, client, &stream).await?;
+async fn diff_stream_and_batch_with_sqls(
```

Comment:
Logic is refactored so we can use it later, when running pre_generated differential test queries.